its/src/test/java/com/sonar/it/csharp/MetricsTest.java

```diff
@@ -33,6 +37,9 @@
 import static com.sonar.it.csharp.Tests.getMeasureAsInt;
 import static org.apache.commons.lang.StringUtils.countMatches;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+import static org.junit.Assume.assumeFalse;
+import static org.junit.Assume.assumeTrue;
```

Comment:
You bet! `Control Alt o` in Linux, might be the same in Windows. You can search for actions using `Control Shift a` (this one would be "organize imports"). One caveat, if you haven't configured your IntelliJ settings as per the [docs](https://github.com/SonarSource/sonar-developer-toolset), then the organize imports action might also rearrange the imports, introducing "junk" changes. Another fairly easy way to remove unused imports is to jump to the top of the file (`Control Home`), then `F2` to jump to the next warning (which should be the first unused import), press `Control x` to delete the line, press `F2` again, and so on ;-)